exercises/concept/paulas-palindromes/.docs/instructions.md

```diff
@@ -0,0 +1,41 @@
+# Instructions
+
+Your friend Paula is trying to get better at solving crossword clues. She has read that [some clues show that the answer is a palindrome][palindrome-crossword-clues].
+A palindrome is a word or sentence that reads the same backards as forwards.
+
+She asks you write her a very simple website that will check whether a string is a palindrome (or not) so that she can check her guesses.
+
+You decide to do this using an Elm [sandbox][browser-sandox] application.
+
+The application will have a Text Box, that Paula can type her guess into, and text stating whether or not the text in the box is a palindrome or not.
+The text will update whenever the text in the Text Box changes.
+
+## 1. Define the Model and Msg types for the application
+
+The application `Model` needs the string to check.
+
+The application needs a `Msg` to indicate that the text in the Text Box has changed.
+The variant for this should be called `Change`
```

Comment:
There is a big issue at this point. After defining the model and message, the tests do not run because tasks 4 and 5 make them crash. That means that nothing will pass until task 5 is finished.

Let's shuffle things around so that the stub file compiles and the tests fail. I'm not sure exactly how to do that, but moving task 4 as task 1 would make sense since it's about creating the default model, it's a good place to start.

There are also issues with using types in the tests that are not defined yet, it might be better to provide things like `type Model = TODO`.

Not sure yet what to do with task 5, maybe removing it, or pre-filling it somehow with some `Debug.todo` in a way that doesn't make the tests crash.

Reply:
Hi Jeremie, I've made some changes to fix this. I think its better this way, but you could debate it. And this certainly wouldn't be the first exercise on exercism to have this problem.

Reply:
Nicely done 💯 

It still doesn't compile before the first task is achieved, but I can't think of a way to test the update function separately if we don't know the type variants, so it's a good compromise.